### PR TITLE
added bg music and sfx for shooting + explosions

### LINKED ISSUE #17: There should be sound effects.
When the gun fires, when a missile explodes, etc.

Pygame's official site put together a nice collection of free stuff to use: https://www.pygame.org/wiki/resources

### LINKED ISSUE #33: Need music to play when playing the main game
This site seems very useful for that: https://freemusicarchive.org/

### PR DESCRIPTION
**Pull request by Rw552**
I just did some sample sfx, we can add different sounds if you find better ones/want different ones!

### CODE REVIEW

File: spinny_gun.py [hunk 1]
```diff
@@ -33,6 +33,14 @@
 MEDIUM_TEXT = pygame.font.Font("freesansbold.ttf", 30)
 SMALL_TEXT = pygame.font.Font("freesansbold.ttf", 20)
 
+# Sound Effects
+shootFx = pygame.mixer.Sound(
+    "assets/audio/135885__andre-onate__gama-laser.wav"
```
Comment (chwerth) [inline spinny_gun.py]: Can we rename this to something shorter?

File: spinny_gun.py [hunk 2]
```diff
@@ -33,6 +33,14 @@
...
+)
+explosionFx = pygame.mixer.Sound(
+    "assets/audio/448226__inspectorj__explosion-8-bit-01.wav"
```
Comment (chwerth) [inline spinny_gun.py]: Same thing here.

File: spinny_gun.py [hunk 3]
```diff
@@ -289,6 +297,7 @@ def game_menu():
 
             if event.type == pygame.KEYUP:
                 if event.key == pygame.K_SPACE:
+                    pygame.mixer.Sound.play(shootFx)
```
Comment (chwerth) [inline spinny_gun.py]: When I tried it, these sound effects sounded a little bit delayed. Not sure why exactly. Maybe the sound could be played after every other statement? Like projectiles.append, or whatever happens to be in the `if` block

File: spinny_gun.py [hunk 4]
```diff
@@ -33,6 +33,14 @@
 MEDIUM_TEXT = pygame.font.Font("freesansbold.ttf", 30)
 SMALL_TEXT = pygame.font.Font("freesansbold.ttf", 20)
 
+# Sound Effects
+shootFx = pygame.mixer.Sound(
```
Comment (chwerth) [inline spinny_gun.py]:
```suggestion
SHOOT_FX = pygame.mixer.Sound(
```

Can we change all these globals to capitalized snake case?

File: spinny_gun.py [hunk 5]
```diff
@@ -33,6 +33,14 @@
...
+    "assets/audio/135885__andre-onate__gama-laser.wav"
+)
+explosionFx = pygame.mixer.Sound(
```
Comment (chwerth) [inline spinny_gun.py]:
```suggestion
EXPLOSION_FX = pygame.mixer.Sound(
```

**Review by chwerth — CHANGES_REQUESTED** [review]
I love all your sound effect / music choices. They're great. Just got a few misc comments

File: spinny_gun.py (hunk 2 above)
Comment (Rw552) [reply spinny_gun.py]: I left the file names titled like that for ease of attribution, but I'll shorten them cause I made a Credits.txt

**Comment by chwerth** [comment]
Would any of these statements work? https://stackoverflow.com/questions/18273722/pygame-sound-delay

**Review by chwerth — APPROVED** [review]

**Pull request merged**